pkg/sdk/go/channels.go

```diff
@@ -76,8 +76,13 @@ func (sdk mfSDK) CreateChannels(chs []Channel, token string) ([]Channel, error)
 	return ccr.Channels, nil
 }
 
-func (sdk mfSDK) Channels(token string, offset, limit uint64, name string) (ChannelsPage, error) {
-	url := fmt.Sprintf("%s/%s?offset=%d&limit=%d&name=%s", sdk.thingsURL, channelsEndpoint, offset, limit, name)
+func (sdk mfSDK) Channels(token string, filter Filter) (ChannelsPage, error) {
+	endpoint, err := sdk.parseFilteredValues(filter)
```

Comment:
`endpoint` naming is confusing here. You are creating the query parameters part of the URL. Probably the best is to change the method to accept `sdk.thingsURL` and `channelsEndpoint`. For example `sdk.createURL(sdk.thingsURL, channelsEndpoint,, filters)`

Reply:
Done